ddtrace/_monkey.py

```diff
@@ -5,13 +5,15 @@
 
 from ddtrace.vendor.wrapt.importer import when_imported
 
-from .internal import telemetry
 from .internal.logger import get_logger
 from .internal.utils import formats
 from .settings import _config as config
 from .settings.asm import config as asm_config
 
 
+if config._telemetry_enabled:
+    from .internal import telemetry
```

Comment:
nit: We can move this import to where it's used (`on_import` function)

Reply:
yeah that's way better

Reply:
well hmmm, I guess technically it's never needed unless the requested module can't be imported. I could load it at the top of `on_import` if you'd rather, but in some cases we may import it and never use it.

Up to you.

Reply:
No issues here. I just wanted to move the import to where it was used. This way we check if telemetry is enabled in two places instead of 3. Also I can see a world where telemetry is enabled via remote config. If that were to happen with this proposal then we would get an import error.

Reply:
Okay done!